package.json

```diff
@@ -11,6 +11,7 @@
   "dependencies": {
     "@mdx-js/loader": "^1.6.21",
     "@next/mdx": "^10.0.2",
+    "axios": "^0.21.0",
```

Comment:
I tried to use `fetch()` data first but encountered some issues. Can we try rewriting it someday?